Investigate the `AssertionError` in the `agent_default_group_added` test

While running the system test `test_cluster/test_agent_groups/test_agent_default_group_added.py` for the 4.4 release, 
it was noticed that this test was failing with the output:

```python
============================= test session starts ==============================
platform linux -- Python 3.9.13, pytest-6.2.2, py-1.10.0, pluggy-0.13.1 -- /home/nstefani/.pyenv/versions/3.9.13/envs/qa-system/bin/python3.9
cachedir: .pytest_cache
metadata: {'Python': '3.9.13', 'Platform': 'Linux-6.0.6-76060006-generic-x86_64-with-glibc2.35', 'Packages': {'pytest': '6.2.2', 'pluggy': '0.13.1'}, 'Plugins': {'html': '3.1.1', 'testinfra': '5.0.0', 'metadata': '2.0.4'}}
rootdir: /home/nstefani/git/wazuh-qa/tests, configfile: pytest.ini
plugins: html-3.1.1, testinfra-5.0.0, metadata-2.0.4
collecting ... collected 3 items

test_cluster/test_agent_groups/test_agent_default_group_added.py::test_agent_default_group_added[wazuh-master-host_manager0-test_infra_agents0-test_infra_managers0] PASSED
test_cluster/test_agent_groups/test_agent_default_group_added.py::test_agent_default_group_added[wazuh-worker1-host_manager0-test_infra_agents0-test_infra_managers0] FAILED
test_cluster/test_agent_groups/test_agent_default_group_added.py::test_agent_default_group_added[wazuh-worker2-host_manager0-test_infra_agents0-test_infra_managers0] FAILED

=================================== FAILURES ===================================
_ test_agent_default_group_added[wazuh-worker1-host_manager0-test_infra_agents0-test_infra_managers0] _

agent_target = 'wazuh-worker1', clean_environment = None
test_infra_managers = ['wazuh-master', 'wazuh-worker1', 'wazuh-worker2']
test_infra_agents = ['wazuh-agent1']
host_manager = <wazuh_testing.tools.system.HostManager object at 0x7fb23a146fa0>

    @pytest.mark.parametrize("test_infra_managers", [test_infra_managers])
    @pytest.mark.parametrize("test_infra_agents", [test_infra_agents])
    @pytest.mark.parametrize("host_manager", [host_manager])
    @pytest.mark.parametrize("agent_target", test_infra_managers)
    def test_agent_default_group_added(agent_target, clean_environment, test_infra_managers,
                                       test_infra_agents, host_manager):
        '''
        description: Check agent enrollment process and default group assignment works as expected in a cluster enviroment.
        An agent pointing to a master/worker node is registered using cli tool, and it gets assigned the default group
        after it is restarted.
        wazuh_min_version: 4.4.0
        parameters:
            - agent_target:
                type: String
                brief: Name of the host where the agent will register.
            - clean_enviroment:
                type: Fixture
                brief: Reset the wazuh log files at the start of the test. Remove all registered agents from master.
            - test_infra_managers
                type: List
                brief: List of manager hosts in enviroment.
            - test_infra_managers
                type: List
                brief: List of agent hosts in enviroment.
            - host_manager
                type: HostManager object
                brief: Handles connection the enviroment's hosts.
        assertions:
            - Verify that after registering the agent key file exists in all nodes.
            - Verify that after registering the agent appears as never_connected in all nodes.
            - Verify that after registering and before starting the agent, it has no groups assigned.
            - Verify that after registering the agent appears as active in all nodes.
            - Verify that after registering and after starting the agent, it has the default group assigned.
        expected_output:
            - f"{agent_id}  {agent_name}  {agent_ip}  never_connected"
            - f"{agent_id}  {agent_name}  {agent_ip}  active"
        '''
    
        agent_ip, agent_id, agent_name, manager_ip = register_agent(test_infra_agents[0], agent_target, host_manager)
    
        # Check that agent status is never-connected in cluster
        time.sleep(timeout)
        check_agent_status(agent_id, agent_name, agent_ip, AGENT_STATUS_NEVER_CONNECTED, host_manager, test_infra_managers)
        # Check that agent has no group assigned
        check_agent_groups(agent_id, AGENT_NO_GROUPS, test_infra_managers, host_manager)
    
        # Check that agent has client key file
        for host in test_infra_agents + test_infra_managers:
            assert check_keys_file(host, host_manager)
    
        # Start the enrollment process by restarting cluster
        restart_cluster(test_infra_agents, host_manager)
        time.sleep(timeout)
    
        # Check if the agent is active in master and workers
        check_agent_status(agent_id, agent_name, agent_ip, AGENT_STATUS_ACTIVE, host_manager, test_infra_managers)
    
        # Check that agent has group set to default
>       check_agent_groups(agent_id, AGENT_GROUPS_DEFAULT, test_infra_managers, host_manager)

test_cluster/test_agent_groups/test_agent_default_group_added.py:125: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

agent_id = '002', group_to_check = 'default'
hosts_list = ['wazuh-master', 'wazuh-worker1', 'wazuh-worker2']
host_manager = <wazuh_testing.tools.system.HostManager object at 0x7fb23a146fa0>

    def check_agent_groups(agent_id, group_to_check, hosts_list, host_manager):
        # Check the expected group is in the group data for the agent
        for host in hosts_list:
            group_data = host_manager.run_command(host, f'{WAZUH_PATH}/bin/agent_groups -s -i {agent_id}')
>           assert group_to_check in group_data, f"Did not recieve expected agent group: {group_to_check} in data \
                                                   {str(group_data)} in host {host}"
E           AssertionError: Did not recieve expected agent group: default in data                                                The agent 'Agent-1669917442' with ID '002' belongs to groups: Null. in host wazuh-master

__init__.py:95: AssertionError

_ test_agent_default_group_added[wazuh-worker2-host_manager0-test_infra_agents0-test_infra_managers0] _

agent_target = 'wazuh-worker2', clean_environment = None
test_infra_managers = ['wazuh-master', 'wazuh-worker1', 'wazuh-worker2']
test_infra_agents = ['wazuh-agent1']
host_manager = <wazuh_testing.tools.system.HostManager object at 0x7fb23a146fa0>

    @pytest.mark.parametrize("test_infra_managers", [test_infra_managers])
    @pytest.mark.parametrize("test_infra_agents", [test_infra_agents])
    @pytest.mark.parametrize("host_manager", [host_manager])
    @pytest.mark.parametrize("agent_target", test_infra_managers)
    def test_agent_default_group_added(agent_target, clean_environment, test_infra_managers,
                                       test_infra_agents, host_manager):
        '''
        description: Check agent enrollment process and default group assignment works as expected in a cluster enviroment.
        An agent pointing to a master/worker node is registered using cli tool, and it gets assigned the default group
        after it is restarted.
        wazuh_min_version: 4.4.0
        parameters:
            - agent_target:
                type: String
                brief: Name of the host where the agent will register.
            - clean_enviroment:
                type: Fixture
                brief: Reset the wazuh log files at the start of the test. Remove all registered agents from master.
            - test_infra_managers
                type: List
                brief: List of manager hosts in enviroment.
            - test_infra_managers
                type: List
                brief: List of agent hosts in enviroment.
            - host_manager
                type: HostManager object
                brief: Handles connection the enviroment's hosts.
        assertions:
            - Verify that after registering the agent key file exists in all nodes.
            - Verify that after registering the agent appears as never_connected in all nodes.
            - Verify that after registering and before starting the agent, it has no groups assigned.
            - Verify that after registering the agent appears as active in all nodes.
            - Verify that after registering and after starting the agent, it has the default group assigned.
        expected_output:
            - f"{agent_id}  {agent_name}  {agent_ip}  never_connected"
            - f"{agent_id}  {agent_name}  {agent_ip}  active"
        '''
    
        agent_ip, agent_id, agent_name, manager_ip = register_agent(test_infra_agents[0], agent_target, host_manager)
    
        # Check that agent status is never-connected in cluster
        time.sleep(timeout)
        check_agent_status(agent_id, agent_name, agent_ip, AGENT_STATUS_NEVER_CONNECTED, host_manager, test_infra_managers)
        # Check that agent has no group assigned
        check_agent_groups(agent_id, AGENT_NO_GROUPS, test_infra_managers, host_manager)
    
        # Check that agent has client key file
        for host in test_infra_agents + test_infra_managers:
            assert check_keys_file(host, host_manager)
    
        # Start the enrollment process by restarting cluster
        restart_cluster(test_infra_agents, host_manager)
        time.sleep(timeout)
    
        # Check if the agent is active in master and workers
        check_agent_status(agent_id, agent_name, agent_ip, AGENT_STATUS_ACTIVE, host_manager, test_infra_managers)
    
        # Check that agent has group set to default
>       check_agent_groups(agent_id, AGENT_GROUPS_DEFAULT, test_infra_managers, host_manager)

test_cluster/test_agent_groups/test_agent_default_group_added.py:125: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

agent_id = '003', group_to_check = 'default'
hosts_list = ['wazuh-master', 'wazuh-worker1', 'wazuh-worker2']
host_manager = <wazuh_testing.tools.system.HostManager object at 0x7fb23a146fa0>

    def check_agent_groups(agent_id, group_to_check, hosts_list, host_manager):
        # Check the expected group is in the group data for the agent
        for host in hosts_list:
            group_data = host_manager.run_command(host, f'{WAZUH_PATH}/bin/agent_groups -s -i {agent_id}')
>           assert group_to_check in group_data, f"Did not recieve expected agent group: {group_to_check} in data \
                                                   {str(group_data)} in host {host}"
E           AssertionError: Did not recieve expected agent group: default in data                                                The agent 'Agent-1669917528' with ID '003' belongs to groups: Null. in host wazuh-master

__init__.py:95: AssertionError

=========================== short test summary info ============================
FAILED test_cluster/test_agent_groups/test_agent_default_group_added.py::test_agent_default_group_added[wazuh-worker1-host_manager0-test_infra_agents0-test_infra_managers0]
FAILED test_cluster/test_agent_groups/test_agent_default_group_added.py::test_agent_default_group_added[wazuh-worker2-host_manager0-test_infra_agents0-test_infra_managers0]
=================== 2 failed, 1 passed in 260.51s (0:04:20) ====================
```
These assertion errors were previously reported in #3285 too.

Output continuation 

```python
_ test_agent_default_group_added[wazuh-worker2-host_manager0-test_infra_agents0-test_infra_managers0] _

agent_target = 'wazuh-worker2', clean_environment = None
test_infra_managers = ['wazuh-master', 'wazuh-worker1', 'wazuh-worker2']
test_infra_agents = ['wazuh-agent1']
host_manager = <wazuh_testing.tools.system.HostManager object at 0x7fb23a146fa0>

    @pytest.mark.parametrize("test_infra_managers", [test_infra_managers])
    @pytest.mark.parametrize("test_infra_agents", [test_infra_agents])
    @pytest.mark.parametrize("host_manager", [host_manager])
    @pytest.mark.parametrize("agent_target", test_infra_managers)
    def test_agent_default_group_added(agent_target, clean_environment, test_infra_managers,
                                       test_infra_agents, host_manager):
        '''
        description: Check agent enrollment process and default group assignment works as expected in a cluster enviroment.
        An agent pointing to a master/worker node is registered using cli tool, and it gets assigned the default group
        after it is restarted.
        wazuh_min_version: 4.4.0
        parameters:
            - agent_target:
                type: String
                brief: Name of the host where the agent will register.
            - clean_enviroment:
                type: Fixture
                brief: Reset the wazuh log files at the start of the test. Remove all registered agents from master.
            - test_infra_managers
                type: List
                brief: List of manager hosts in enviroment.
            - test_infra_managers
                type: List
                brief: List of agent hosts in enviroment.
            - host_manager
                type: HostManager object
                brief: Handles connection the enviroment's hosts.
        assertions:
            - Verify that after registering the agent key file exists in all nodes.
            - Verify that after registering the agent appears as never_connected in all nodes.
            - Verify that after registering and before starting the agent, it has no groups assigned.
            - Verify that after registering the agent appears as active in all nodes.
            - Verify that after registering and after starting the agent, it has the default group assigned.
        expected_output:
            - f"{agent_id}  {agent_name}  {agent_ip}  never_connected"
            - f"{agent_id}  {agent_name}  {agent_ip}  active"
        '''
    
        agent_ip, agent_id, agent_name, manager_ip = register_agent(test_infra_agents[0], agent_target, host_manager)
    
        # Check that agent status is never-connected in cluster
        time.sleep(timeout)
        check_agent_status(agent_id, agent_name, agent_ip, AGENT_STATUS_NEVER_CONNECTED, host_manager, test_infra_managers)
        # Check that agent has no group assigned
        check_agent_groups(agent_id, AGENT_NO_GROUPS, test_infra_managers, host_manager)
    
        # Check that agent has client key file
        for host in test_infra_agents + test_infra_managers:
            assert check_keys_file(host, host_manager)
    
        # Start the enrollment process by restarting cluster
        restart_cluster(test_infra_agents, host_manager)
        time.sleep(timeout)
    
        # Check if the agent is active in master and workers
        check_agent_status(agent_id, agent_name, agent_ip, AGENT_STATUS_ACTIVE, host_manager, test_infra_managers)
    
        # Check that agent has group set to default
>       check_agent_groups(agent_id, AGENT_GROUPS_DEFAULT, test_infra_managers, host_manager)

test_cluster/test_agent_groups/test_agent_default_group_added.py:125: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

agent_id = '003', group_to_check = 'default'
hosts_list = ['wazuh-master', 'wazuh-worker1', 'wazuh-worker2']
host_manager = <wazuh_testing.tools.system.HostManager object at 0x7fb23a146fa0>

    def check_agent_groups(agent_id, group_to_check, hosts_list, host_manager):
        # Check the expected group is in the group data for the agent
        for host in hosts_list:
            group_data = host_manager.run_command(host, f'{WAZUH_PATH}/bin/agent_groups -s -i {agent_id}')
>           assert group_to_check in group_data, f"Did not recieve expected agent group: {group_to_check} in data \
                                                   {str(group_data)} in host {host}"
E           AssertionError: Did not recieve expected agent group: default in data                                                The agent 'Agent-1669917528' with ID '003' belongs to groups: Null. in host wazuh-master

__init__.py:95: AssertionError
------------------------------ Captured log setup ------------------------------
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b'ansible --tree /tmp/tmpcyuvmba4 -i /home/nstefani/git/wazuh-qa/tests/system/provisioning/enrollment_cluster/inventory.yml -m copy --args \'dest=/var/ossec/logs/ossec.log content=\'"\'"\'\'"\'"\' force=yes\' wazuh-agent1', exit_status=0, stdout=b'wazuh-agent1 | CHANGED => {\n    "changed": true,\n    "checksum": "da39a3ee5e6b4b0d3255bfef95601890afd80709",\n    "dest": "/var/ossec/logs/ossec.log",\n    "gid": 999,\n    "group": "wazuh",\n    "md5sum": "d41d8cd98f00b204e9800998ecf8427e",\n    "mode": "0660",\n    "owner": "wazuh",\n    "size": 0,\n    "src": "/root/.ansible/tmp/ansible-tmp-1669917510.6006641-1424983-218317605806778/source",\n    "state": "file",\n    "uid": 999\n}\n', stderr=None)
INFO     testinfra:ansible.py:58 RUN Ansible('copy', "dest=/var/ossec/logs/ossec.log content='' force=yes", {'check': False, 'become': False}): {'changed': True,
 'checksum': 'da39a3ee5e6b4b0d3255bfef95601890afd80709',
 'dest': '/var/ossec/logs/ossec.log',
 'gid': 999,
 'group': 'wazuh',
 'md5sum': 'd41d8cd98f00b204e9800998ecf8427e',
 'mode': '0660',
 'owner': 'wazuh',
 'size': 0,
 'src': '/root/.ansible/tmp/ansible-tmp-1669917510.6006641-1424983-218317605806778/source',
 'state': 'file',
 'uid': 999}
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b'ansible --tree /tmp/tmp33gpm9zw -i /home/nstefani/git/wazuh-qa/tests/system/provisioning/enrollment_cluster/inventory.yml -m copy --args \'dest=/var/ossec/logs/ossec.log content=\'"\'"\'\'"\'"\' force=yes\' wazuh-master', exit_status=0, stdout=b'wazuh-master | CHANGED => {\n    "changed": true,\n    "checksum": "da39a3ee5e6b4b0d3255bfef95601890afd80709",\n    "dest": "/var/ossec/logs/ossec.log",\n    "gid": 999,\n    "group": "wazuh",\n    "md5sum": "d41d8cd98f00b204e9800998ecf8427e",\n    "mode": "0660",\n    "owner": "wazuh",\n    "size": 0,\n    "src": "/root/.ansible/tmp/ansible-tmp-1669917513.0387325-1425320-121294711909859/source",\n    "state": "file",\n    "uid": 999\n}\n', stderr=None)
INFO     testinfra:ansible.py:58 RUN Ansible('copy', "dest=/var/ossec/logs/ossec.log content='' force=yes", {'check': False, 'become': False}): {'changed': True,
 'checksum': 'da39a3ee5e6b4b0d3255bfef95601890afd80709',
 'dest': '/var/ossec/logs/ossec.log',
 'gid': 999,
 'group': 'wazuh',
 'md5sum': 'd41d8cd98f00b204e9800998ecf8427e',
 'mode': '0660',
 'owner': 'wazuh',
 'size': 0,
 'src': '/root/.ansible/tmp/ansible-tmp-1669917513.0387325-1425320-121294711909859/source',
 'state': 'file',
 'uid': 999}
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b'ansible --tree /tmp/tmpaqflnnmb -i /home/nstefani/git/wazuh-qa/tests/system/provisioning/enrollment_cluster/inventory.yml -m copy --args \'dest=/var/ossec/logs/cluster.log content=\'"\'"\'\'"\'"\' force=yes\' wazuh-master', exit_status=0, stdout=b'wazuh-master | SUCCESS => {\n    "changed": false,\n    "checksum": "da39a3ee5e6b4b0d3255bfef95601890afd80709",\n    "dest": "/var/ossec/logs/cluster.log",\n    "gid": 999,\n    "group": "wazuh",\n    "mode": "0660",\n    "owner": "wazuh",\n    "path": "/var/ossec/logs/cluster.log",\n    "size": 0,\n    "state": "file",\n    "uid": 999\n}\n', stderr=None)
INFO     testinfra:ansible.py:58 RUN Ansible('copy', "dest=/var/ossec/logs/cluster.log content='' force=yes", {'check': False, 'become': False}): {'changed': False,
 'checksum': 'da39a3ee5e6b4b0d3255bfef95601890afd80709',
 'dest': '/var/ossec/logs/cluster.log',
 'gid': 999,
 'group': 'wazuh',
 'mode': '0660',
 'owner': 'wazuh',
 'path': '/var/ossec/logs/cluster.log',
 'size': 0,
 'state': 'file',
 'uid': 999}
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b'ansible --tree /tmp/tmp283o8qok -i /home/nstefani/git/wazuh-qa/tests/system/provisioning/enrollment_cluster/inventory.yml -m copy --args \'dest=/var/ossec/logs/ossec.log content=\'"\'"\'\'"\'"\' force=yes\' wazuh-worker1', exit_status=0, stdout=b'wazuh-worker1 | CHANGED => {\n    "changed": true,\n    "checksum": "da39a3ee5e6b4b0d3255bfef95601890afd80709",\n    "dest": "/var/ossec/logs/ossec.log",\n    "gid": 999,\n    "group": "wazuh",\n    "md5sum": "d41d8cd98f00b204e9800998ecf8427e",\n    "mode": "0660",\n    "owner": "wazuh",\n    "size": 0,\n    "src": "/root/.ansible/tmp/ansible-tmp-1669917517.5589168-1425945-139677853105721/source",\n    "state": "file",\n    "uid": 999\n}\n', stderr=None)
INFO     testinfra:ansible.py:58 RUN Ansible('copy', "dest=/var/ossec/logs/ossec.log content='' force=yes", {'check': False, 'become': False}): {'changed': True,
 'checksum': 'da39a3ee5e6b4b0d3255bfef95601890afd80709',
 'dest': '/var/ossec/logs/ossec.log',
 'gid': 999,
 'group': 'wazuh',
 'md5sum': 'd41d8cd98f00b204e9800998ecf8427e',
 'mode': '0660',
 'owner': 'wazuh',
 'size': 0,
 'src': '/root/.ansible/tmp/ansible-tmp-1669917517.5589168-1425945-139677853105721/source',
 'state': 'file',
 'uid': 999}
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b'ansible --tree /tmp/tmp4llgr1eq -i /home/nstefani/git/wazuh-qa/tests/system/provisioning/enrollment_cluster/inventory.yml -m copy --args \'dest=/var/ossec/logs/cluster.log content=\'"\'"\'\'"\'"\' force=yes\' wazuh-worker1', exit_status=0, stdout=b'wazuh-worker1 | SUCCESS => {\n    "changed": false,\n    "checksum": "da39a3ee5e6b4b0d3255bfef95601890afd80709",\n    "dest": "/var/ossec/logs/cluster.log",\n    "gid": 999,\n    "group": "wazuh",\n    "mode": "0660",\n    "owner": "wazuh",\n    "path": "/var/ossec/logs/cluster.log",\n    "size": 0,\n    "state": "file",\n    "uid": 999\n}\n', stderr=None)
INFO     testinfra:ansible.py:58 RUN Ansible('copy', "dest=/var/ossec/logs/cluster.log content='' force=yes", {'check': False, 'become': False}): {'changed': False,
 'checksum': 'da39a3ee5e6b4b0d3255bfef95601890afd80709',
 'dest': '/var/ossec/logs/cluster.log',
 'gid': 999,
 'group': 'wazuh',
 'mode': '0660',
 'owner': 'wazuh',
 'path': '/var/ossec/logs/cluster.log',
 'size': 0,
 'state': 'file',
 'uid': 999}
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b'ansible --tree /tmp/tmpqjwm4xp6 -i /home/nstefani/git/wazuh-qa/tests/system/provisioning/enrollment_cluster/inventory.yml -m copy --args \'dest=/var/ossec/logs/ossec.log content=\'"\'"\'\'"\'"\' force=yes\' wazuh-worker2', exit_status=0, stdout=b'wazuh-worker2 | CHANGED => {\n    "changed": true,\n    "checksum": "da39a3ee5e6b4b0d3255bfef95601890afd80709",\n    "dest": "/var/ossec/logs/ossec.log",\n    "gid": 999,\n    "group": "wazuh",\n    "md5sum": "d41d8cd98f00b204e9800998ecf8427e",\n    "mode": "0660",\n    "owner": "wazuh",\n    "size": 0,\n    "src": "/root/.ansible/tmp/ansible-tmp-1669917522.3275018-1426579-24761637180890/source",\n    "state": "file",\n    "uid": 999\n}\n', stderr=None)
INFO     testinfra:ansible.py:58 RUN Ansible('copy', "dest=/var/ossec/logs/ossec.log content='' force=yes", {'check': False, 'become': False}): {'changed': True,
 'checksum': 'da39a3ee5e6b4b0d3255bfef95601890afd80709',
 'dest': '/var/ossec/logs/ossec.log',
 'gid': 999,
 'group': 'wazuh',
 'md5sum': 'd41d8cd98f00b204e9800998ecf8427e',
 'mode': '0660',
 'owner': 'wazuh',
 'size': 0,
 'src': '/root/.ansible/tmp/ansible-tmp-1669917522.3275018-1426579-24761637180890/source',
 'state': 'file',
 'uid': 999}
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b'ansible --tree /tmp/tmpc0o6zmal -i /home/nstefani/git/wazuh-qa/tests/system/provisioning/enrollment_cluster/inventory.yml -m copy --args \'dest=/var/ossec/logs/cluster.log content=\'"\'"\'\'"\'"\' force=yes\' wazuh-worker2', exit_status=0, stdout=b'wazuh-worker2 | SUCCESS => {\n    "changed": false,\n    "checksum": "da39a3ee5e6b4b0d3255bfef95601890afd80709",\n    "dest": "/var/ossec/logs/cluster.log",\n    "gid": 999,\n    "group": "wazuh",\n    "mode": "0660",\n    "owner": "wazuh",\n    "path": "/var/ossec/logs/cluster.log",\n    "size": 0,\n    "state": "file",\n    "uid": 999\n}\n', stderr=None)
INFO     testinfra:ansible.py:58 RUN Ansible('copy', "dest=/var/ossec/logs/cluster.log content='' force=yes", {'check': False, 'become': False}): {'changed': False,
 'checksum': 'da39a3ee5e6b4b0d3255bfef95601890afd80709',
 'dest': '/var/ossec/logs/cluster.log',
 'gid': 999,
 'group': 'wazuh',
 'mode': '0660',
 'owner': 'wazuh',
 'path': '/var/ossec/logs/cluster.log',
 'size': 0,
 'state': 'file',
 'uid': 999}
------------------------------ Captured log call -------------------------------
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b"ansible --tree /tmp/tmp2zsacwm3 -i /home/nstefani/git/wazuh-qa/tests/system/provisioning/enrollment_cluster/inventory.yml -m command --args 'hostname -i' wazuh-agent1", exit_status=0, stdout=b'wazuh-agent1 | CHANGED | rc=0 >>\n172.28.0.5\n', stderr=None)
INFO     testinfra:ansible.py:58 RUN Ansible('command', 'hostname -i', {'check': False, 'become': False}): {'changed': True,
 'cmd': ['hostname', '-i'],
 'delta': '0:00:00.108966',
 'end': '2022-12-01 18:58:47.634502',
 'msg': '',
 'rc': 0,
 'start': '2022-12-01 18:58:47.525536',
 'stderr': '',
 'stderr_lines': [],
 'stdout': '172.28.0.5',
 'stdout_lines': ['172.28.0.5']}
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b"ansible --tree /tmp/tmplq6lspmt -i /home/nstefani/git/wazuh-qa/tests/system/provisioning/enrollment_cluster/inventory.yml -m command --args 'hostname -i' wazuh-worker2", exit_status=0, stdout=b'wazuh-worker2 | CHANGED | rc=0 >>\n172.28.0.4\n', stderr=None)
INFO     testinfra:ansible.py:58 RUN Ansible('command', 'hostname -i', {'check': False, 'become': False}): {'changed': True,
 'cmd': ['hostname', '-i'],
 'delta': '0:00:00.107735',
 'end': '2022-12-01 18:58:49.236418',
 'msg': '',
 'rc': 0,
 'start': '2022-12-01 18:58:49.128683',
 'stderr': '',
 'stderr_lines': [],
 'stdout': '172.28.0.4',
 'stdout_lines': ['172.28.0.4']}
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b'ansible --tree /tmp/tmph61a9x_6 -i /home/nstefani/git/wazuh-qa/tests/system/provisioning/enrollment_cluster/inventory.yml -m replace --args \'path=/var/ossec/etc/ossec.conf regexp=\'"\'"\'<address>[\\s\\S]+</address>\'"\'"\' replace=\'"\'"\'<address>172.28.0.4</address>\'"\'"\'\' wazuh-agent1', exit_status=0, stdout=b'wazuh-agent1 | CHANGED => {\n    "changed": true,\n    "msg": "1 replacements made",\n    "rc": 0\n}\n', stderr=None)
INFO     testinfra:ansible.py:58 RUN Ansible('replace', "path=/var/ossec/etc/ossec.conf regexp='<address>[\\s\\S]+</address>' replace='<address>172.28.0.4</address>'", {'check': False, 'become': False}): {'changed': True, 'msg': '1 replacements made', 'rc': 0}
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b"ansible --tree /tmp/tmpjhhdzoxe -i /home/nstefani/git/wazuh-qa/tests/system/provisioning/enrollment_cluster/inventory.yml -m command --args '/var/ossec/bin/agent-auth -m 172.28.0.4 -A Agent-1669917528 -I 172.28.0.5' wazuh-agent1", exit_status=0, stdout=b'wazuh-agent1 | CHANGED | rc=0 >>\n2022/12/01 18:58:52 agent-auth: INFO: Started (pid: 10893).\n2022/12/01 18:58:52 agent-auth: INFO: Requesting a key from server: 172.28.0.4\n2022/12/01 18:58:52 agent-auth: INFO: No authentication password provided\n2022/12/01 18:58:52 agent-auth: INFO: Using agent name as: Agent-1669917528\n2022/12/01 18:58:52 agent-auth: INFO: Waiting for server reply\n2022/12/01 18:58:52 agent-auth: INFO: Valid key received\n', stderr=None)
INFO     testinfra:ansible.py:58 RUN Ansible('command', '/var/ossec/bin/agent-auth -m 172.28.0.4 -A Agent-1669917528 -I 172.28.0.5', {'check': False, 'become': False}): {'changed': True,
 'cmd': ['/var/ossec/bin/agent-auth',
         '-m',
         '172.28.0.4',
         '-A',
         'Agent-1669917528',
         '-I',
         '172.28.0.5'],
 'delta': '0:00:00.124206',
 'end': '2022-12-01 18:58:52.231865',
 'msg': '',
 'rc': 0,
 'start': '2022-12-01 18:58:52.107659',
 'stderr': '2022/12/01 18:58:52 agent-auth: INFO: Started (pid: 10893).\n'
           '2022/12/01 18:58:52 agent-auth: INFO: Requesting a key from '
           'server: 172.28.0.4\n'
           '2022/12/01 18:58:52 agent-auth: INFO: No authentication password '
           'provided\n'
           '2022/12/01 18:58:52 agent-auth: INFO: Using agent name as: '
           'Agent-1669917528\n'
           '2022/12/01 18:58:52 agent-auth: INFO: Waiting for server reply\n'
           '2022/12/01 18:58:52 agent-auth: INFO: Valid key received',
 'stderr_lines': ['2022/12/01 18:58:52 agent-auth: INFO: Started (pid: 10893).',
                  '2022/12/01 18:58:52 agent-auth: INFO: Requesting a key from '
                  'server: 172.28.0.4',
                  '2022/12/01 18:58:52 agent-auth: INFO: No authentication '
                  'password provided',
                  '2022/12/01 18:58:52 agent-auth: INFO: Using agent name as: '
                  'Agent-1669917528',
                  '2022/12/01 18:58:52 agent-auth: INFO: Waiting for server '
                  'reply',
                  '2022/12/01 18:58:52 agent-auth: INFO: Valid key received'],
 'stdout': '',
 'stdout_lines': []}
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b"ansible --tree /tmp/tmpeknft4k5 -i /home/nstefani/git/wazuh-qa/tests/system/provisioning/enrollment_cluster/inventory.yml -m command --args 'cut -c 1-3 /var/ossec/etc/client.keys' wazuh-agent1", exit_status=0, stdout=b'wazuh-agent1 | CHANGED | rc=0 >>\n003\n', stderr=None)
INFO     testinfra:ansible.py:58 RUN Ansible('command', 'cut -c 1-3 /var/ossec/etc/client.keys', {'check': False, 'become': False}): {'changed': True,
 'cmd': ['cut', '-c', '1-3', '/var/ossec/etc/client.keys'],
 'delta': '0:00:00.106974',
 'end': '2022-12-01 18:58:53.708492',
 'msg': '',
 'rc': 0,
 'start': '2022-12-01 18:58:53.601518',
 'stderr': '',
 'stderr_lines': [],
 'stdout': '003',
 'stdout_lines': ['003']}
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b"ansible --tree /tmp/tmpdbm36jqo -i /home/nstefani/git/wazuh-qa/tests/system/provisioning/enrollment_cluster/inventory.yml -m command --args '/var/ossec/bin/cluster_control -a' wazuh-master", exit_status=0, stdout=b'wazuh-master | CHANGED | rc=0 >>\nID   NAME              IP          STATUS           VERSION       NODE NAME     \n000  wazuh-master      127.0.0.1   active           Wazuh v4.4.0  wazuh-master  \n003  Agent-1669917528  172.28.0.5  never_connected  unknown       unknown       \n', stderr=None)
INFO     testinfra:ansible.py:58 RUN Ansible('command', '/var/ossec/bin/cluster_control -a', {'check': False, 'become': False}): {'changed': True,
 'cmd': ['/var/ossec/bin/cluster_control', '-a'],
 'delta': '0:00:00.331093',
 'end': '2022-12-01 18:59:00.429577',
 'msg': '',
 'rc': 0,
 'start': '2022-12-01 18:59:00.098484',
 'stderr': '',
 'stderr_lines': [],
 'stdout': 'ID   NAME              IP          STATUS           VERSION       '
           'NODE NAME     \n'
           '000  wazuh-master      127.0.0.1   active           Wazuh v4.4.0  '
           'wazuh-master  \n'
           '003  Agent-1669917528  172.28.0.5  never_connected  unknown       '
           'unknown       ',
 'stdout_lines': ['ID   NAME              IP          STATUS           '
                  'VERSION       NODE NAME     ',
                  '000  wazuh-master      127.0.0.1   active           Wazuh '
                  'v4.4.0  wazuh-master  ',
                  '003  Agent-1669917528  172.28.0.5  never_connected  '
                  'unknown       unknown       ']}
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b"ansible --tree /tmp/tmpgvuw0hbj -i /home/nstefani/git/wazuh-qa/tests/system/provisioning/enrollment_cluster/inventory.yml -m command --args '/var/ossec/bin/cluster_control -a' wazuh-worker1", exit_status=0, stdout=b'wazuh-worker1 | CHANGED | rc=0 >>\nID   NAME              IP          STATUS           VERSION       NODE NAME     \n000  wazuh-master      127.0.0.1   active           Wazuh v4.4.0  wazuh-master  \n003  Agent-1669917528  172.28.0.5  never_connected  unknown       unknown       \n', stderr=None)
INFO     testinfra:ansible.py:58 RUN Ansible('command', '/var/ossec/bin/cluster_control -a', {'check': False, 'become': False}): {'changed': True,
 'cmd': ['/var/ossec/bin/cluster_control', '-a'],
 'delta': '0:00:00.368931',
 'end': '2022-12-01 18:59:02.195911',
 'msg': '',
 'rc': 0,
 'start': '2022-12-01 18:59:01.826980',
 'stderr': '',
 'stderr_lines': [],
 'stdout': 'ID   NAME              IP          STATUS           VERSION       '
           'NODE NAME     \n'
           '000  wazuh-master      127.0.0.1   active           Wazuh v4.4.0  '
           'wazuh-master  \n'
           '003  Agent-1669917528  172.28.0.5  never_connected  unknown       '
           'unknown       ',
 'stdout_lines': ['ID   NAME              IP          STATUS           '
                  'VERSION       NODE NAME     ',
                  '000  wazuh-master      127.0.0.1   active           Wazuh '
                  'v4.4.0  wazuh-master  ',
                  '003  Agent-1669917528  172.28.0.5  never_connected  '
                  'unknown       unknown       ']}
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b"ansible --tree /tmp/tmpgswhmlhy -i /home/nstefani/git/wazuh-qa/tests/system/provisioning/enrollment_cluster/inventory.yml -m command --args '/var/ossec/bin/cluster_control -a' wazuh-worker2", exit_status=0, stdout=b'wazuh-worker2 | CHANGED | rc=0 >>\nID   NAME              IP          STATUS           VERSION       NODE NAME     \n000  wazuh-master      127.0.0.1   active           Wazuh v4.4.0  wazuh-master  \n003  Agent-1669917528  172.28.0.5  never_connected  unknown       unknown       \n', stderr=None)
INFO     testinfra:ansible.py:58 RUN Ansible('command', '/var/ossec/bin/cluster_control -a', {'check': False, 'become': False}): {'changed': True,
 'cmd': ['/var/ossec/bin/cluster_control', '-a'],
 'delta': '0:00:00.336985',
 'end': '2022-12-01 18:59:03.962777',
 'msg': '',
 'rc': 0,
 'start': '2022-12-01 18:59:03.625792',
 'stderr': '',
 'stderr_lines': [],
 'stdout': 'ID   NAME              IP          STATUS           VERSION       '
           'NODE NAME     \n'
           '000  wazuh-master      127.0.0.1   active           Wazuh v4.4.0  '
           'wazuh-master  \n'
           '003  Agent-1669917528  172.28.0.5  never_connected  unknown       '
           'unknown       ',
 'stdout_lines': ['ID   NAME              IP          STATUS           '
                  'VERSION       NODE NAME     ',
                  '000  wazuh-master      127.0.0.1   active           Wazuh '
                  'v4.4.0  wazuh-master  ',
                  '003  Agent-1669917528  172.28.0.5  never_connected  '
                  'unknown       unknown       ']}
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b"ansible --tree /tmp/tmpe1l45sgx -i /home/nstefani/git/wazuh-qa/tests/system/provisioning/enrollment_cluster/inventory.yml -m command --args '/var/ossec/bin/agent_groups -s -i 003' wazuh-master", exit_status=0, stdout=b"wazuh-master | CHANGED | rc=0 >>\nThe agent 'Agent-1669917528' with ID '003' belongs to groups: Null.\n", stderr=None)
INFO     testinfra:ansible.py:58 RUN Ansible('command', '/var/ossec/bin/agent_groups -s -i 003', {'check': False, 'become': False}): {'changed': True,
 'cmd': ['/var/ossec/bin/agent_groups', '-s', '-i', '003'],
 'delta': '0:00:00.530202',
 'end': '2022-12-01 18:59:06.013674',
 'msg': '',
 'rc': 0,
 'start': '2022-12-01 18:59:05.483472',
 'stderr': '',
 'stderr_lines': [],
 'stdout': "The agent 'Agent-1669917528' with ID '003' belongs to groups: "
           'Null.',
 'stdout_lines': ["The agent 'Agent-1669917528' with ID '003' belongs to "
                  'groups: Null.']}
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b"ansible --tree /tmp/tmprwg8_vzf -i /home/nstefani/git/wazuh-qa/tests/system/provisioning/enrollment_cluster/inventory.yml -m command --args '/var/ossec/bin/agent_groups -s -i 003' wazuh-worker1", exit_status=0, stdout=b"wazuh-worker1 | CHANGED | rc=0 >>\nThe agent 'Agent-1669917528' with ID '003' belongs to groups: Null.\n", stderr=None)
INFO     testinfra:ansible.py:58 RUN Ansible('command', '/var/ossec/bin/agent_groups -s -i 003', {'check': False, 'become': False}): {'changed': True,
 'cmd': ['/var/ossec/bin/agent_groups', '-s', '-i', '003'],
 'delta': '0:00:00.551265',
 'end': '2022-12-01 18:59:07.944848',
 'msg': '',
 'rc': 0,
 'start': '2022-12-01 18:59:07.393583',
 'stderr': '',
 'stderr_lines': [],
 'stdout': "The agent 'Agent-1669917528' with ID '003' belongs to groups: "
           'Null.',
 'stdout_lines': ["The agent 'Agent-1669917528' with ID '003' belongs to "
                  'groups: Null.']}
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b"ansible --tree /tmp/tmpuguashps -i /home/nstefani/git/wazuh-qa/tests/system/provisioning/enrollment_cluster/inventory.yml -m command --args '/var/ossec/bin/agent_groups -s -i 003' wazuh-worker2", exit_status=0, stdout=b"wazuh-worker2 | CHANGED | rc=0 >>\nThe agent 'Agent-1669917528' with ID '003' belongs to groups: Null.\n", stderr=None)
INFO     testinfra:ansible.py:58 RUN Ansible('command', '/var/ossec/bin/agent_groups -s -i 003', {'check': False, 'become': False}): {'changed': True,
 'cmd': ['/var/ossec/bin/agent_groups', '-s', '-i', '003'],
 'delta': '0:00:00.522709',
 'end': '2022-12-01 18:59:09.935624',
 'msg': '',
 'rc': 0,
 'start': '2022-12-01 18:59:09.412915',
 'stderr': '',
 'stderr_lines': [],
 'stdout': "The agent 'Agent-1669917528' with ID '003' belongs to groups: "
           'Null.',
 'stdout_lines': ["The agent 'Agent-1669917528' with ID '003' belongs to "
                  'groups: Null.']}
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b"docker exec wazuh-agent1 /bin/sh -c 'cat -- /var/ossec/etc/client.keys'", exit_status=0, stdout=b'003 Agent-1669917528 172.28.0.5 15f96770486f3da9f2d576892a4c7c50e86fa719ac3667512b9cad8ff8d33445\n', stderr=None)
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b"docker exec wazuh-master /bin/sh -c 'cat -- /var/ossec/etc/client.keys'", exit_status=0, stdout=b'003 Agent-1669917528 172.28.0.5 15f96770486f3da9f2d576892a4c7c50e86fa719ac3667512b9cad8ff8d33445\n', stderr=None)
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b"docker exec wazuh-worker1 /bin/sh -c 'cat -- /var/ossec/etc/client.keys'", exit_status=0, stdout=b'003 Agent-1669917528 172.28.0.5 15f96770486f3da9f2d576892a4c7c50e86fa719ac3667512b9cad8ff8d33445\n', stderr=None)
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b"docker exec wazuh-worker2 /bin/sh -c 'cat -- /var/ossec/etc/client.keys'", exit_status=0, stdout=b'003 Agent-1669917528 172.28.0.5 15f96770486f3da9f2d576892a4c7c50e86fa719ac3667512b9cad8ff8d33445\n', stderr=None)
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b"ansible --tree /tmp/tmp0_a_sfbb -i /home/nstefani/git/wazuh-qa/tests/system/provisioning/enrollment_cluster/inventory.yml -m command --args 'service wazuh-agent restart' wazuh-agent1", exit_status=0, stdout=b'wazuh-agent1 | CHANGED | rc=0 >>\nwazuh-modulesd not running...\nwazuh-logcollector not running...\nwazuh-syscheckd not running...\nwazuh-agentd not running...\nwazuh-execd not running...\nWazuh v4.4.0 Stopped\nStarting Wazuh v4.4.0...\nStarted wazuh-execd...\nStarted wazuh-agentd...\nStarted wazuh-syscheckd...\nStarted wazuh-logcollector...\nStarted wazuh-modulesd...\nCompleted.2022/12/01 18:59:11 wazuh-agentd[11049] debug_op.c:70 at _log(): DEBUG: Logging module auto-initialized\n2022/12/01 18:59:11 wazuh-agentd[11049] main.c:139 at main(): DEBUG: Wazuh home directory: /var/ossec\n2022/12/01 18:59:11 wazuh-agentd[11049] main.c:141 at main(): DEBUG: Started (pid: 11049).\n2022/12/01 18:59:12 wazuh-agentd[11049] agent_op.c:219 at os_read_agent_profile(): DEBUG: Calling os_read_agent_profile().\n2022/12/01 18:59:12 wazuh-agentd[11049] agent_op.c:238 at os_read_agent_profile(): DEBUG: os_read_agent_profile() = [debian, debian10]\n2022/12/01 18:59:12 wazuh-agentd[11049] config.c:424 at ReadConfig(): DEBUG: agent_config element does not have any attributes.\n2022/12/01 18:59:12 wazuh-agentd[11049] agent_op.c:219 at os_read_agent_profile(): DEBUG: Calling os_read_agent_profile().\n2022/12/01 18:59:12 wazuh-agentd[11049] agent_op.c:238 at os_read_agent_profile(): DEBUG: os_read_agent_profile() = [debian, debian10]\n2022/12/01 18:59:12 wazuh-agentd[11049] config.c:424 at ReadConfig(): DEBUG: agent_config element does not have any attributes.\n2022/12/01 18:59:13 wazuh-agentd[11072] debug_op.c:70 at _log(): DEBUG: Logging module auto-initialized\n2022/12/01 18:59:13 wazuh-agentd[11072] main.c:139 at main(): DEBUG: Wazuh home directory: /var/ossec\n2022/12/01 18:59:13 wazuh-agentd[11072] main.c:141 at main(): DEBUG: Started (pid: 11072).\n2022/12/01 18:59:13 wazuh-agentd[11072] agent_op.c:219 at os_read_agent_profile(): DEBUG: Calling os_read_agent_profile().\n2022/12/01 18:59:13 wazuh-agentd[11072] agent_op.c:238 at os_read_agent_profile(): DEBUG: os_read_agent_profile() = [debian, debian10]\n2022/12/01 18:59:13 wazuh-agentd[11072] config.c:424 at ReadConfig(): DEBUG: agent_config element does not have any attributes.\n2022/12/01 18:59:13 wazuh-agentd[11072] agent_op.c:219 at os_read_agent_profile(): DEBUG: Calling os_read_agent_profile().\n2022/12/01 18:59:13 wazuh-agentd[11072] agent_op.c:238 at os_read_agent_profile(): DEBUG: os_read_agent_profile() = [debian, debian10]\n2022/12/01 18:59:13 wazuh-agentd[11072] config.c:424 at ReadConfig(): DEBUG: agent_config element does not have any attributes.\n', stderr=None)
INFO     testinfra:ansible.py:58 RUN Ansible('command', 'service wazuh-agent restart', {'check': False, 'become': False}): {'changed': True,
 'cmd': ['service', 'wazuh-agent', 'restart'],
 'delta': '0:00:07.214455',
 'end': '2022-12-01 18:59:19.087184',
 'msg': '',
 'rc': 0,
 'start': '2022-12-01 18:59:11.872729',
 'stderr': '2022/12/01 18:59:11 wazuh-agentd[11049] debug_op.c:70 at _log(): '
           'DEBUG: Logging module auto-initialized\n'
           '2022/12/01 18:59:11 wazuh-agentd[11049] main.c:139 at main(): '
           'DEBUG: Wazuh home directory: /var/ossec\n'
           '2022/12/01 18:59:11 wazuh-agentd[11049] main.c:141 at main(): '
           'DEBUG: Started (pid: 11049).\n'
           '2022/12/01 18:59:12 wazuh-agentd[11049] agent_op.c:219 at '
           'os_read_agent_profile(): DEBUG: Calling os_read_agent_profile().\n'
           '2022/12/01 18:59:12 wazuh-agentd[11049] agent_op.c:238 at '
           'os_read_agent_profile(): DEBUG: os_read_agent_profile() = [debian, '
           'debian10]\n'
           '2022/12/01 18:59:12 wazuh-agentd[11049] config.c:424 at '
           'ReadConfig(): DEBUG: agent_config element does not have any '
           'attributes.\n'
           '2022/12/01 18:59:12 wazuh-agentd[11049] agent_op.c:219 at '
           'os_read_agent_profile(): DEBUG: Calling os_read_agent_profile().\n'
           '2022/12/01 18:59:12 wazuh-agentd[11049] agent_op.c:238 at '
           'os_read_agent_profile(): DEBUG: os_read_agent_profile() = [debian, '
           'debian10]\n'
           '2022/12/01 18:59:12 wazuh-agentd[11049] config.c:424 at '
           'ReadConfig(): DEBUG: agent_config element does not have any '
           'attributes.\n'
           '2022/12/01 18:59:13 wazuh-agentd[11072] debug_op.c:70 at _log(): '
           'DEBUG: Logging module auto-initialized\n'
           '2022/12/01 18:59:13 wazuh-agentd[11072] main.c:139 at main(): '
           'DEBUG: Wazuh home directory: /var/ossec\n'
           '2022/12/01 18:59:13 wazuh-agentd[11072] main.c:141 at main(): '
           'DEBUG: Started (pid: 11072).\n'
           '2022/12/01 18:59:13 wazuh-agentd[11072] agent_op.c:219 at '
           'os_read_agent_profile(): DEBUG: Calling os_read_agent_profile().\n'
           '2022/12/01 18:59:13 wazuh-agentd[11072] agent_op.c:238 at '
           'os_read_agent_profile(): DEBUG: os_read_agent_profile() = [debian, '
           'debian10]\n'
           '2022/12/01 18:59:13 wazuh-agentd[11072] config.c:424 at '
           'ReadConfig(): DEBUG: agent_config element does not have any '
           'attributes.\n'
           '2022/12/01 18:59:13 wazuh-agentd[11072] agent_op.c:219 at '
           'os_read_agent_profile(): DEBUG: Calling os_read_agent_profile().\n'
           '2022/12/01 18:59:13 wazuh-agentd[11072] agent_op.c:238 at '
           'os_read_agent_profile(): DEBUG: os_read_agent_profile() = [debian, '
           'debian10]\n'
           '2022/12/01 18:59:13 wazuh-agentd[11072] config.c:424 at '
           'ReadConfig(): DEBUG: agent_config element does not have any '
           'attributes.',
 'stderr_lines': ['2022/12/01 18:59:11 wazuh-agentd[11049] debug_op.c:70 at '
                  '_log(): DEBUG: Logging module auto-initialized',
                  '2022/12/01 18:59:11 wazuh-agentd[11049] main.c:139 at '
                  'main(): DEBUG: Wazuh home directory: /var/ossec',
                  '2022/12/01 18:59:11 wazuh-agentd[11049] main.c:141 at '
                  'main(): DEBUG: Started (pid: 11049).',
                  '2022/12/01 18:59:12 wazuh-agentd[11049] agent_op.c:219 at '
                  'os_read_agent_profile(): DEBUG: Calling '
                  'os_read_agent_profile().',
                  '2022/12/01 18:59:12 wazuh-agentd[11049] agent_op.c:238 at '
                  'os_read_agent_profile(): DEBUG: os_read_agent_profile() = '
                  '[debian, debian10]',
                  '2022/12/01 18:59:12 wazuh-agentd[11049] config.c:424 at '
                  'ReadConfig(): DEBUG: agent_config element does not have any '
                  'attributes.',
                  '2022/12/01 18:59:12 wazuh-agentd[11049] agent_op.c:219 at '
                  'os_read_agent_profile(): DEBUG: Calling '
                  'os_read_agent_profile().',
                  '2022/12/01 18:59:12 wazuh-agentd[11049] agent_op.c:238 at '
                  'os_read_agent_profile(): DEBUG: os_read_agent_profile() = '
                  '[debian, debian10]',
                  '2022/12/01 18:59:12 wazuh-agentd[11049] config.c:424 at '
                  'ReadConfig(): DEBUG: agent_config element does not have any '
                  'attributes.',
                  '2022/12/01 18:59:13 wazuh-agentd[11072] debug_op.c:70 at '
                  '_log(): DEBUG: Logging module auto-initialized',
                  '2022/12/01 18:59:13 wazuh-agentd[11072] main.c:139 at '
                  'main(): DEBUG: Wazuh home directory: /var/ossec',
                  '2022/12/01 18:59:13 wazuh-agentd[11072] main.c:141 at '
                  'main(): DEBUG: Started (pid: 11072).',
                  '2022/12/01 18:59:13 wazuh-agentd[11072] agent_op.c:219 at '
                  'os_read_agent_profile(): DEBUG: Calling '
                  'os_read_agent_profile().',
                  '2022/12/01 18:59:13 wazuh-agentd[11072] agent_op.c:238 at '
                  'os_read_agent_profile(): DEBUG: os_read_agent_profile() = '
                  '[debian, debian10]',
                  '2022/12/01 18:59:13 wazuh-agentd[11072] config.c:424 at '
                  'ReadConfig(): DEBUG: agent_config element does not have any '
                  'attributes.',
                  '2022/12/01 18:59:13 wazuh-agentd[11072] agent_op.c:219 at '
                  'os_read_agent_profile(): DEBUG: Calling '
                  'os_read_agent_profile().',
                  '2022/12/01 18:59:13 wazuh-agentd[11072] agent_op.c:238 at '
                  'os_read_agent_profile(): DEBUG: os_read_agent_profile() = '
                  '[debian, debian10]',
                  '2022/12/01 18:59:13 wazuh-agentd[11072] config.c:424 at '
                  'ReadConfig(): DEBUG: agent_config element does not have any '
                  'attributes.'],
 'stdout': 'wazuh-modulesd not running...\n'
           'wazuh-logcollector not running...\n'
           'wazuh-syscheckd not running...\n'
           'wazuh-agentd not running...\n'
           'wazuh-execd not running...\n'
           'Wazuh v4.4.0 Stopped\n'
           'Starting Wazuh v4.4.0...\n'
           'Started wazuh-execd...\n'
           'Started wazuh-agentd...\n'
           'Started wazuh-syscheckd...\n'
           'Started wazuh-logcollector...\n'
           'Started wazuh-modulesd...\n'
           'Completed.',
 'stdout_lines': ['wazuh-modulesd not running...',
                  'wazuh-logcollector not running...',
                  'wazuh-syscheckd not running...',
                  'wazuh-agentd not running...',
                  'wazuh-execd not running...',
                  'Wazuh v4.4.0 Stopped',
                  'Starting Wazuh v4.4.0...',
                  'Started wazuh-execd...',
                  'Started wazuh-agentd...',
                  'Started wazuh-syscheckd...',
                  'Started wazuh-logcollector...',
                  'Started wazuh-modulesd...',
                  'Completed.']}
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b"ansible --tree /tmp/tmpi1th_5an -i /home/nstefani/git/wazuh-qa/tests/system/provisioning/enrollment_cluster/inventory.yml -m service --args 'name=wazuh-agent state=restarted' wazuh-agent1", exit_status=0, stdout=b'wazuh-agent1 | CHANGED => {\n    "changed": true,\n    "name": "wazuh-agent",\n    "state": "started"\n}\n', stderr=None)
INFO     testinfra:ansible.py:58 RUN Ansible('service', 'name=wazuh-agent state=restarted', {'check': False, 'become': False}): {'changed': True, 'name': 'wazuh-agent', 'state': 'started'}
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b"ansible --tree /tmp/tmp_jofxhrc -i /home/nstefani/git/wazuh-qa/tests/system/provisioning/enrollment_cluster/inventory.yml -m command --args '/var/ossec/bin/cluster_control -a' wazuh-master", exit_status=0, stdout=b'wazuh-master | CHANGED | rc=0 >>\nID   NAME              IP          STATUS  VERSION       NODE NAME      \n000  wazuh-master      127.0.0.1   active  Wazuh v4.4.0  wazuh-master   \n003  Agent-1669917528  172.28.0.5  active  Wazuh v4.4.0  wazuh-worker2  \n', stderr=None)
INFO     testinfra:ansible.py:58 RUN Ansible('command', '/var/ossec/bin/cluster_control -a', {'check': False, 'become': False}): {'changed': True,
 'cmd': ['/var/ossec/bin/cluster_control', '-a'],
 'delta': '0:00:00.340144',
 'end': '2022-12-01 18:59:35.198287',
 'msg': '',
 'rc': 0,
 'start': '2022-12-01 18:59:34.858143',
 'stderr': '',
 'stderr_lines': [],
 'stdout': 'ID   NAME              IP          STATUS  VERSION       NODE '
           'NAME      \n'
           '000  wazuh-master      127.0.0.1   active  Wazuh v4.4.0  '
           'wazuh-master   \n'
           '003  Agent-1669917528  172.28.0.5  active  Wazuh v4.4.0  '
           'wazuh-worker2  ',
 'stdout_lines': ['ID   NAME              IP          STATUS  VERSION       '
                  'NODE NAME      ',
                  '000  wazuh-master      127.0.0.1   active  Wazuh v4.4.0  '
                  'wazuh-master   ',
                  '003  Agent-1669917528  172.28.0.5  active  Wazuh v4.4.0  '
                  'wazuh-worker2  ']}
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b"ansible --tree /tmp/tmp44e1xrts -i /home/nstefani/git/wazuh-qa/tests/system/provisioning/enrollment_cluster/inventory.yml -m command --args '/var/ossec/bin/cluster_control -a' wazuh-worker1", exit_status=0, stdout=b'wazuh-worker1 | CHANGED | rc=0 >>\nID   NAME              IP          STATUS  VERSION       NODE NAME      \n000  wazuh-master      127.0.0.1   active  Wazuh v4.4.0  wazuh-master   \n003  Agent-1669917528  172.28.0.5  active  Wazuh v4.4.0  wazuh-worker2  \n', stderr=None)
INFO     testinfra:ansible.py:58 RUN Ansible('command', '/var/ossec/bin/cluster_control -a', {'check': False, 'become': False}): {'changed': True,
 'cmd': ['/var/ossec/bin/cluster_control', '-a'],
 'delta': '0:00:00.369559',
 'end': '2022-12-01 18:59:37.036451',
 'msg': '',
 'rc': 0,
 'start': '2022-12-01 18:59:36.666892',
 'stderr': '',
 'stderr_lines': [],
 'stdout': 'ID   NAME              IP          STATUS  VERSION       NODE '
           'NAME      \n'
           '000  wazuh-master      127.0.0.1   active  Wazuh v4.4.0  '
           'wazuh-master   \n'
           '003  Agent-1669917528  172.28.0.5  active  Wazuh v4.4.0  '
           'wazuh-worker2  ',
 'stdout_lines': ['ID   NAME              IP          STATUS  VERSION       '
                  'NODE NAME      ',
                  '000  wazuh-master      127.0.0.1   active  Wazuh v4.4.0  '
                  'wazuh-master   ',
                  '003  Agent-1669917528  172.28.0.5  active  Wazuh v4.4.0  '
                  'wazuh-worker2  ']}
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b"ansible --tree /tmp/tmpm8dt6dsb -i /home/nstefani/git/wazuh-qa/tests/system/provisioning/enrollment_cluster/inventory.yml -m command --args '/var/ossec/bin/cluster_control -a' wazuh-worker2", exit_status=0, stdout=b'wazuh-worker2 | CHANGED | rc=0 >>\nID   NAME              IP          STATUS  VERSION       NODE NAME      \n000  wazuh-master      127.0.0.1   active  Wazuh v4.4.0  wazuh-master   \n003  Agent-1669917528  172.28.0.5  active  Wazuh v4.4.0  wazuh-worker2  \n', stderr=None)
INFO     testinfra:ansible.py:58 RUN Ansible('command', '/var/ossec/bin/cluster_control -a', {'check': False, 'become': False}): {'changed': True,
 'cmd': ['/var/ossec/bin/cluster_control', '-a'],
 'delta': '0:00:00.335728',
 'end': '2022-12-01 18:59:38.756013',
 'msg': '',
 'rc': 0,
 'start': '2022-12-01 18:59:38.420285',
 'stderr': '',
 'stderr_lines': [],
 'stdout': 'ID   NAME              IP          STATUS  VERSION       NODE '
           'NAME      \n'
           '000  wazuh-master      127.0.0.1   active  Wazuh v4.4.0  '
           'wazuh-master   \n'
           '003  Agent-1669917528  172.28.0.5  active  Wazuh v4.4.0  '
           'wazuh-worker2  ',
 'stdout_lines': ['ID   NAME              IP          STATUS  VERSION       '
                  'NODE NAME      ',
                  '000  wazuh-master      127.0.0.1   active  Wazuh v4.4.0  '
                  'wazuh-master   ',
                  '003  Agent-1669917528  172.28.0.5  active  Wazuh v4.4.0  '
                  'wazuh-worker2  ']}
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b"ansible --tree /tmp/tmp7ryc01bl -i /home/nstefani/git/wazuh-qa/tests/system/provisioning/enrollment_cluster/inventory.yml -m command --args '/var/ossec/bin/agent_groups -s -i 003' wazuh-master", exit_status=0, stdout=b"wazuh-master | CHANGED | rc=0 >>\nThe agent 'Agent-1669917528' with ID '003' belongs to groups: Null.\n", stderr=None)
INFO     testinfra:ansible.py:58 RUN Ansible('command', '/var/ossec/bin/agent_groups -s -i 003', {'check': False, 'become': False}): {'changed': True,
 'cmd': ['/var/ossec/bin/agent_groups', '-s', '-i', '003'],
 'delta': '0:00:00.533288',
 'end': '2022-12-01 18:59:40.737437',
 'msg': '',
 'rc': 0,
 'start': '2022-12-01 18:59:40.204149',
 'stderr': '',
 'stderr_lines': [],
 'stdout': "The agent 'Agent-1669917528' with ID '003' belongs to groups: "
           'Null.',
 'stdout_lines': ["The agent 'Agent-1669917528' with ID '003' belongs to "
                  'groups: Null.']}
---------------------------- Captured log teardown -----------------------------
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b"ansible --tree /tmp/tmp6zq_rxm7 -i /home/nstefani/git/wazuh-qa/tests/system/provisioning/enrollment_cluster/inventory.yml -m command --args 'cut -c 1-3 /var/ossec/etc/client.keys' wazuh-master", exit_status=0, stdout=b'wazuh-master | CHANGED | rc=0 >>\n003\n', stderr=None)
INFO     testinfra:ansible.py:58 RUN Ansible('command', 'cut -c 1-3 /var/ossec/etc/client.keys', {'check': False, 'become': False}): {'changed': True,
 'cmd': ['cut', '-c', '1-3', '/var/ossec/etc/client.keys'],
 'delta': '0:00:00.106310',
 'end': '2022-12-01 18:59:42.292721',
 'msg': '',
 'rc': 0,
 'start': '2022-12-01 18:59:42.186411',
 'stderr': '',
 'stderr_lines': [],
 'stdout': '003',
 'stdout_lines': ['003']}
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b"ansible --tree /tmp/tmpbr109n0j -i /home/nstefani/git/wazuh-qa/tests/system/provisioning/enrollment_cluster/inventory.yml -m service --args 'name=wazuh-agent state=stopped' wazuh-agent1", exit_status=0, stdout=b'wazuh-agent1 | CHANGED => {\n    "changed": true,\n    "name": "wazuh-agent",\n    "state": "stopped"\n}\n', stderr=None)
INFO     testinfra:ansible.py:58 RUN Ansible('service', 'name=wazuh-agent state=stopped', {'check': False, 'become': False}): {'changed': True, 'name': 'wazuh-agent', 'state': 'stopped'}
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b'ansible --tree /tmp/tmp7dpck4mh -i /home/nstefani/git/wazuh-qa/tests/system/provisioning/enrollment_cluster/inventory.yml -m copy --args \'dest=/var/ossec/etc/client.keys content=\'"\'"\'\'"\'"\' force=yes\' wazuh-agent1', exit_status=0, stdout=b'wazuh-agent1 | CHANGED => {\n    "changed": true,\n    "checksum": "da39a3ee5e6b4b0d3255bfef95601890afd80709",\n    "dest": "/var/ossec/etc/client.keys",\n    "gid": 999,\n    "group": "wazuh",\n    "md5sum": "d41d8cd98f00b204e9800998ecf8427e",\n    "mode": "0640",\n    "owner": "root",\n    "size": 0,\n    "src": "/root/.ansible/tmp/ansible-tmp-1669917586.334106-1432862-232193568416774/source",\n    "state": "file",\n    "uid": 0\n}\n', stderr=None)
INFO     testinfra:ansible.py:58 RUN Ansible('copy', "dest=/var/ossec/etc/client.keys content='' force=yes", {'check': False, 'become': False}): {'changed': True,
 'checksum': 'da39a3ee5e6b4b0d3255bfef95601890afd80709',
 'dest': '/var/ossec/etc/client.keys',
 'gid': 999,
 'group': 'wazuh',
 'md5sum': 'd41d8cd98f00b204e9800998ecf8427e',
 'mode': '0640',
 'owner': 'root',
 'size': 0,
 'src': '/root/.ansible/tmp/ansible-tmp-1669917586.334106-1432862-232193568416774/source',
 'state': 'file',
 'uid': 0}
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b"ansible --tree /tmp/tmp4losc3zc -i /home/nstefani/git/wazuh-qa/tests/system/provisioning/enrollment_cluster/inventory.yml -m command --args '/var/ossec/bin/manage_agents -r 003' wazuh-master", exit_status=0, stdout=b"wazuh-master | CHANGED | rc=0 >>\n\n\n****************************************\n* Wazuh v4.4.0 Agent manager.          *\n* The following options are available: *\n****************************************\n   (A)dd an agent (A).\n   (E)xtract key for an agent (E).\n   (L)ist already added agents (L).\n   (R)emove an agent (R).\n   (Q)uit.\nChoose your action: A,E,L,R or Q: \nAvailable agents: \n   ID: 003, Name: Agent-1669917528, IP: 172.28.0.5\nProvide the ID of the agent to be removed (or '\\q' to quit): 003\nConfirm deleting it?(y/n): y\nAgent '003' removed.\n\nmanage_agents: Exiting.\n", stderr=None)
INFO     testinfra:ansible.py:58 RUN Ansible('command', '/var/ossec/bin/manage_agents -r 003', {'check': False, 'become': False}): {'changed': True,
 'cmd': ['/var/ossec/bin/manage_agents', '-r', '003'],
 'delta': '0:00:00.116296',
 'end': '2022-12-01 18:59:49.436545',
 'msg': '',
 'rc': 0,
 'start': '2022-12-01 18:59:49.320249',
 'stderr': '',
 'stderr_lines': [],
 'stdout': '\n'
           '\n'
           '****************************************\n'
           '* Wazuh v4.4.0 Agent manager.          *\n'
           '* The following options are available: *\n'
           '****************************************\n'
           '   (A)dd an agent (A).\n'
           '   (E)xtract key for an agent (E).\n'
           '   (L)ist already added agents (L).\n'
           '   (R)emove an agent (R).\n'
           '   (Q)uit.\n'
           'Choose your action: A,E,L,R or Q: \n'
           'Available agents: \n'
           '   ID: 003, Name: Agent-1669917528, IP: 172.28.0.5\n'
           "Provide the ID of the agent to be removed (or '\\q' to quit): 003\n"
           'Confirm deleting it?(y/n): y\n'
           "Agent '003' removed.\n"
           '\n'
           'manage_agents: Exiting.',
 'stdout_lines': ['',
                  '',
                  '****************************************',
                  '* Wazuh v4.4.0 Agent manager.          *',
                  '* The following options are available: *',
                  '****************************************',
                  '   (A)dd an agent (A).',
                  '   (E)xtract key for an agent (E).',
                  '   (L)ist already added agents (L).',
                  '   (R)emove an agent (R).',
                  '   (Q)uit.',
                  'Choose your action: A,E,L,R or Q: ',
                  'Available agents: ',
                  '   ID: 003, Name: Agent-1669917528, IP: 172.28.0.5',
                  "Provide the ID of the agent to be removed (or '\\q' to "
                  'quit): 003',
                  'Confirm deleting it?(y/n): y',
                  "Agent '003' removed.",
                  '',
                  'manage_agents: Exiting.']}
DEBUG    testinfra:base.py:277 RUN CommandResult(command=b"ansible --tree /tmp/tmprqmk3goy -i /home/nstefani/git/wazuh-qa/tests/system/provisioning/enrollment_cluster/inventory.yml -m command --args 'cut -c 1-3 /var/ossec/etc/client.keys' wazuh-master", exit_status=0, stdout=b'wazuh-master | CHANGED | rc=0 >>\n\n', stderr=None)
INFO     testinfra:ansible.py:58 RUN Ansible('command', 'cut -c 1-3 /var/ossec/etc/client.keys', {'check': False, 'become': False}): {'changed': True,
 'cmd': ['cut', '-c', '1-3', '/var/ossec/etc/client.keys'],
 'delta': '0:00:00.112444',
 'end': '2022-12-01 18:59:51.016825',
 'msg': '',
 'rc': 0,
 'start': '2022-12-01 18:59:50.904381',
 'stderr': '',
 'stderr_lines': [],
 'stdout': '',
 'stdout_lines': []}
=========================== short test summary info ============================
FAILED test_cluster/test_agent_groups/test_agent_default_group_added.py::test_agent_default_group_added[wazuh-worker1-host_manager0-test_infra_agents0-test_infra_managers0]
FAILED test_cluster/test_agent_groups/test_agent_default_group_added.py::test_agent_default_group_added[wazuh-worker2-host_manager0-test_infra_agents0-test_infra_managers0]
=================== 2 failed, 1 passed in 260.51s (0:04:20) ====================

```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -82,6 +82,7 @@ Release report: TBD
 - Fix version validation in qa-ctl config generator ([#2454](https://github.com/wazuh/wazuh-qa/pull/2454)) \- (Framework)
 - Fix invalid reference for test_api_endpoints_performance.py xfail items ([#3378](https://github.com/wazuh/wazuh-qa/pull/3378)) \- (Tests)
 - Fix undeclared API token variable in multigroups system tests ([#3674](https://github.com/wazuh/wazuh-qa/pull/3674)) \- (Framework + Tests)
+- Fix sleep time in `test_agent_default_group_added`. ([#3692](https://github.com/wazuh/wazuh-qa/pull/3692)) \- (Tests)
 
 ### Removed
 

--- a/tests/system/test_cluster/test_agent_groups/test_agent_default_group_added.py
+++ b/tests/system/test_cluster/test_agent_groups/test_agent_default_group_added.py
@@ -60,7 +60,7 @@ inventory_path = os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(os
 host_manager = HostManager(inventory_path)
 local_path = os.path.dirname(os.path.abspath(__file__))
 tmp_path = os.path.join(local_path, 'tmp')
-timeout = 5
+timeout = 25
 
 
 # Tests

